packages/@fbcmobile-ui/Screens/BugReportScreen.js

```diff
@@ -21,13 +21,12 @@ import React, {useEffect, useRef, useState} from 'react';
 import Spinner from 'react-native-spinkit';
 import TextFormInput from '@fbcmobile/ui/Components/FormInput/TextFormInput';
 import UserActionLogger from '@fbcmobile/ui/Logging/UserActionLogger';
-import fbt from 'fbt';
-import nullthrows from 'nullthrows';
 import {Colors} from '@fbcmobile/ui/Theme';
 import {EVENT} from '@fbcmobile/ui/Logging/UserActionEvents';
 import {Icon} from 'react-native-material-ui';
 import {Image, ScrollView, View} from 'react-native';
 import {TextInput} from 'react-native';
+import {fbt} from 'fbt';
```

Comment:
I don't think we need to import this in brackets, we've always done:

`import fbt from 'fbt';`